lib/src/room.dart

```diff
@@ -1370,6 +1370,21 @@ class Room {
       return getParticipants(membershipFilter);
     }
 
+    return await requestParticipantsFromServer(
+      membershipFilter,
+      suppressWarning,
+      cache,
+    );
+  }
+
+  Future<List<User>> requestParticipantsFromServer(
+      [List<Membership> membershipFilter = const [
+        Membership.join,
+        Membership.invite,
+        Membership.knock,
+      ],
+      bool suppressWarning = false,
+      bool cache = true]) async {
```

Comment:
- Create `const membershipFilter`
- Used to specify named parameters in a function call.
```dart
Future<List<User>> requestParticipantsFromServer({
 membershipFilter,
 bool suppressWarning = false,
 bool cache = true,
});
```

Reply:
done